### ISSUE TITLE
"End-Of-File" error

### ISSUE THREAD

**Issue by ffxsam** [Borewit/tokenizer-s3#266]
Hey @Borewit, long time no see! I'm finally working on my project again, and running into some trouble.

Check out the Node.js test script below. It's yielding the error below. Any ideas? The file is a valid WAV file.

```
Error: End-Of-File
    at RangeRequestTokenizer.loadRange (/Volumes/SuperData/Sites/reelcrafter/v2-upload-processor/node_modules/@tokenizer/range/lib/range-request-tokenizer.js:89:19)
    at RangeRequestTokenizer.peekBuffer (/Volumes/SuperData/Sites/reelcrafter/v2-upload-processor/node_modules/@tokenizer/range/lib/range-request-tokenizer.js:68:21)
    at RangeRequestTokenizer.readBuffer (/Volumes/SuperData/Sites/reelcrafter/v2-upload-processor/node_modules/@tokenizer/range/lib/range-request-tokenizer.js:35:35)
    at RangeRequestTokenizer.readToken (/Volumes/SuperData/Sites/reelcrafter/v2-upload-processor/node_modules/strtok3/lib/AbstractTokenizer.js:25:32)
    at WaveParser.readWaveChunk (/Volumes/SuperData/Sites/reelcrafter/v2-upload-processor/node_modules/music-metadata/lib/riff/WaveParser.js:49:49)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Function._parse (/Volumes/SuperData/Sites/reelcrafter/v2-upload-processor/node_modules/music-metadata/lib/ParserFactory.js:159:9)
    at async /Volumes/SuperData/Sites/reelcrafter/v2-upload-processor/tmp.js:22:16
```

```js
const S3 = require('aws-sdk/clients/s3');
const { makeTokenizer } = require('@tokenizer/s3');
const { parseFromTokenizer } = require('music-metadata/lib/core');

const s3 = new S3({ apiVersion: '2006-03-01' });

(async function () {
  let metadata;

  const s3Tokenizer = await makeTokenizer(
    s3,
    {
      Bucket: 'xx',
      Key: 'xx.wav',
    },
    {
      disableChunked: false,
    },
  );

  try {
    metadata = await parseFromTokenizer(s3Tokenizer, {
      skipCovers: true,
    });
  } catch (e) {
    console.error(e);
  }

  console.log(metadata);
})();
```

**Comment by ffxsam** [comment]
Using `parseBuffer` to read the file locally, it works perfectly.

**Comment by Borewit** [comment]
Hi @ffxsam, good to hear from you.

> Any suggestions here?

Not directly. If I point to a public accessible `wav` it seems to work. 

```js

const S3 = require('aws-sdk/clients/s3');
const { makeTokenizer } = require('@tokenizer/s3');
const { parseFromTokenizer } = require('music-metadata');

const s3 = new S3({ apiVersion: '2006-03-01' });

(async function () {
  let metadata;

  const s3Tokenizer = await makeTokenizer(
    s3,
    {
      Bucket: 'ftmedia',
      Key: 'adigoldstein/august2015/cooking-show-background-music.wav',
    },
    {
      disableChunked: false,
    },
  );

  try {
    metadata = await parseFromTokenizer(s3Tokenizer, {
      skipCovers: true,
    });
  } catch (e) {
    console.error(e);
  }

  console.log(metadata);
})();

```

**Comment by ffxsam** [comment]
This wav file throws an error.

[wow.wav.zip](https://github.com/Borewit/tokenizer-s3/files/5076248/wow.wav.zip)

**Comment by Borewit** [comment]
This issue should be solved with [music-metadata v7.0.2](https://github.com/Borewit/music-metadata/releases/tag/v7.0.2)

**Comment by ffxsam** [comment]
Perfect, thank you!!
